DuckDuckGo/TabViewController.swift

```diff
@@ -1325,16 +1326,9 @@ extension TabViewController: WKNavigationDelegate {
                                                                            navigationAction: navigationAction,
                                                                            onStartExtracting: { showProgressIndicator() },
                                                                            onFinishExtracting: { },
-                                                                           onLinkRewrite: { [weak self] newURL, navigationAction in
+                                                                           onLinkRewrite: { [weak self] newRequest, navigationAction in
                 guard let self = self else { return }
-                if self.isNewTargetBlankRequest(navigationAction: navigationAction) {
-                    self.delegate?.tab(self,
-                                       didRequestNewTabForUrl: newURL,
-                                       openedByPage: true,
-                                       inheritingAttribution: self.adClickAttributionLogic.state)
-                } else {
-                    self.load(url: newURL)
-                }
+                self.load(urlRequest: newRequest)
```

Comment:
I wonder, why remove `isNewTargetBlankRequest` call ?

Reply:
Since we're re-issuing the request its probably better to have the normal navigation policy change handle the new tab creation